src/datadoc/backend/core.py

```diff
@@ -78,6 +78,11 @@ def _set_variable_uuid(self) -> None:
             if v.id is None:
                 v.id = uuid.uuid4()
 
+    def _set_default_value_variable_is_personal_data(self) -> None:
```

Comment:
This could be moved to a utils file in backend, that takes a list and sets a default value to each element. The same applies for _set_variable_uuid.